.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        flavour: [assembleOculusvrArm64World, assembleHvrArm64World, assemblePicoxrArm64World]
+        flavour: [assembleOculusvrArm64WorldGecko, assembleHvrArm64WorldGecko, assemblePicoxrArm64WorldGecko]
```

Comment:
Nit: maybe time to make this array multiline, so it is a bit more version control friendly.